auth-api/src/auth_api/resources/v1/products.py

```diff
@@ -22,12 +22,14 @@
 from auth_api.exceptions import BusinessException
 from auth_api.services import Product as ProductService
 from auth_api.utils.endpoints_enums import EndpointEnum
+from auth_api.utils.auth import jwt as _jwt
 
 bp = Blueprint("PRODUCTS", __name__, url_prefix=f"{EndpointEnum.API_V1.value}/products")
 
 
 @bp.route("", methods=["GET", "OPTIONS"])
 @cross_origin(origins="*", methods=["GET"])
+@_jwt.requires_auth
```

Comment:
was this missing before?

Reply:
I see this might fail?

![image](https://github.com/user-attachments/assets/70860449-b764-4756-b3f8-0be19f9a0bf6)

Reply:
yeah it was missing